.sourcery/AutomaticRuleTests.stencil

```diff
@@ -5,7 +5,7 @@ import XCTest
 
 {% for rule in types.structs %}
 {% if rule.name|hasSuffix:"Rule" %}
-class {{ rule.name }}Tests: XCTestCase {
+class {{ rule.name }}ExamplesTests: XCTestCase {
```

Comment:
Maybe `GeneratedTests` instead? Since the hand-written tests may also use "examples".

```suggestion
class {{ rule.name }}GeneratedTests: XCTestCase {
```

And `AutomaticRuleTests.{stencil,swift}` could be renamed to `GeneratedTests.{stencil,swift}`

Reply:
Good suggestion!